examples/main/src/layer-examples.js

```diff
@@ -122,7 +120,8 @@ const GeoJsonLayer64Example = {
     },
     strokeWidth: 10,
     strokeWidthMinPixels: 1,
-    pickable: true
+    pickable: true,
+    fp64: true
```

Comment:
If we add this to `defaultProps` then it should show up as a checkbox and we'll only need one example per layer?